docs/RELEASE_CHECKLIST.md

```diff
@@ -1,5 +1,23 @@
 # Release checklist
 
+## After hard freeze
+
+- [ ] Bump all CI requirements to current versions:
+
+      ```bash
+      rm requirements-ci-current.txt requirements-ci-fenics.txt conda-lock.yml
+      make ci_requirements
+      ```
+
+      Commit all changes.
+      Before proceeding, wait until changes are merged.
```

Comment:
This should fix the generated HTML. There are three more similar issues below.

```suggestion
- [ ] Bump all CI requirements to current versions:

    ```bash
    rm requirements-ci-current.txt requirements-ci-fenics.txt conda-lock.yml
    make ci_requirements
    ```

    Commit all changes.
    Before proceeding, wait until changes are merged.
```